source/googleAnalytics.brs

```diff
@@ -182,5 +181,13 @@ Function gamobileSendHit(hit_params As String) As Void
 
   ' Increment the cache buster
   m.gamobile.next_z = m.gamobile.next_z + 1
-  
+
+End Function
+
+Function tostr(input_string)
```

Comment:
Looking through the Roku forums, Joel has [indicated](http://forums.roku.com/viewtopic.php?f=34&t=66938&p=466671&hilit=license+examples#p466671) the examples are under the MIT license.